rcsb/utils/tests-seq/testInterProProvider.py

```diff
@@ -43,7 +43,7 @@ def tearDown(self):
         logger.info("Completed %s at %s (%.4f seconds)\n", self.id(), time.strftime("%Y %m %d %H:%M:%S", time.localtime()), endTime - self.__startTime)
 
     def testInterProCache(self):
-        ipP = InterProProvider(cachePath=self.__cachePath, useCache=False)
+        ipP = InterProProvider(cachePath=self.__cachePath, useCache=False, useFallBack=False)
```

Comment:
@piehld does `useFallBack=True` cause azure to fail?

Reply:
The fallback is used by default if fetching from InterPro directly fails, so this test case was passing and thus not notifying us that it failed to fetch directly from InterPro.

So, I added this `useFallBack` argument solely for the purpose of running this test _without_ resorting to the fall back data. By default though, throughout the rest of our codebase, the fall back will be used by if source fetching fails. However, now we at least will get notified on the Sunday CI/CD build if fetching from source is failing.

Reply:
Am I understanding correctly that https://github.com/rcsb/py-rcsb_utils_seq/blob/master/rcsb/utils/tests-seq/testInterProProvider.py#L45 wasn't behaving as expected because it was by default using the fallback when the source failed and therefore replicating the behavior of https://github.com/rcsb/py-rcsb_utils_seq/blob/master/rcsb/utils/tests-seq/testInterProProvider.py#L69?

Reply:
Yes, exactly.

Reply:
Thanks for the clarification.